### PR TITLE
feat(ng-excel-builder): add EntriesKeys to public api

### PR DESCRIPTION
**Pull request by StepaZol**
## PR Checklist

Please check if your PR fulfills the following requirements:

-   [x] The commit message follows our guidelines: https://github.com/ngxs/store/blob/master/CONTRIBUTING.md#commit
-   [ ] Tests for the changes have been added (for bug fixes / features)
-   [ ] Docs have been added / updated (for bug fixes / features)

## PR Type

What kind of change does this PR introduce?

```
[ ] Bugfix
[x] Feature
[ ] Code style update (formatting, local variables)
[ ] Refactoring (no functional changes, no api changes)
[ ] Build related changes
[ ] CI related changes
[ ] Documentation content changes
[ ] Other... Please describe:
```

## What is the current behavior?

Issue Number: N/A

## Does this PR introduce a breaking change?

```
[ ] Yes
[x] No
```

### CODE REVIEW

File: packages/ng-excel-builder/package.json [hunk 1]
```diff
@@ -1,6 +1,6 @@
 {
     "name": "@angular-ru/ng-excel-builder",
-    "version": "15.267.0",
+    "version": "15.268.0",
```
Comment (splincode) [inline packages/ng-excel-builder/package.json:3]: revert please

Comment (Schirbak) [inline packages/ng-excel-builder/package.json:3]: revert it, it bumps automatically

**Review by Schirbak — CHANGES_REQUESTED** [review]

File: packages/ng-excel-builder/package.json (hunk 1 above)
Comment (StepaZol) [reply packages/ng-excel-builder/package.json]: fix

Comment (StepaZol) [reply packages/ng-excel-builder/package.json]: fix

**Review by splincode — APPROVED** [review]

**Review by Schirbak — APPROVED** [review]

**Pull request merged**